test_data/yast/raid1_lvm/lvm_raid1.yaml

```diff
@@ -0,0 +1,12 @@
+---
+arch: x86_64
```

Comment:
you are not using `arch` neither. Remove it from all yamls.

Reply:
done